pkg/apis/virt/v1alpha1/types.go

```diff
@@ -152,11 +168,13 @@ type ContainerRootfsVolumeSource struct {
 }
 
 type PersistentVolumeClaimVolumeSource struct {
-	ClaimName string `json:"claimName"`
+	Hogplugable bool   `json:"hotplugable,omitempty"`
+	ClaimName   string `json:"claimName"`
 }
 
 type DataVolumeVolumeSource struct {
-	VolumeName string `json:"volumeName"`
+	Hogplugable bool   `json:"hotplugable,omitempty"`
```

Comment:
We should mark disk hot pluggable, not volume?

Reply:
as disk and volume is one-one relationship, whether a disk support hotpug depends on volume.  and only PVC and DataVolume volume source support Hotplug, so `Hotpluggable` field is in `PersistentVolumeSource` and `DataVolumeSource` instead of `Volume`. if we put it in disk is not so precise.